docs/UserGuide.adoc

```diff
@@ -374,20 +375,25 @@ You type your flashcards out on excel and later save it in your project root dir
 to import it over to your application.
 
 
-NOTE: The csv file imported should follow the format below
-
-.Sample format for csv file
-image::Blood.csv.png[width = "800"]
+NOTE: The csv file imported should follow the format described below
 
 ====
-* The first row of the csv file should have the following headers, Question, Answer, Options, Hints.
+* The first row of the csv file should have the following headers:
++
+Question, Answer, Options, Hints.
 * Question and Answer are mandatory fields, and should not be left blank.
-* Options can take 0 or many values.
 * Hints can take 0 or 1 values only.
+* For MCQ cards, The csv file only accepts one incorrect answer for each flashcard.
```

Comment:
Should this be "one correct answer"?

Reply:
Ahh i can change that. I was trying to say that u can only put one incorrect answer if you wish to have MCQ flashcards